BUCK

```diff
@@ -405,6 +405,7 @@ cpp_library_wrapper(name="rocksdb_tools_lib", srcs=[
         "test_util/testutil.cc",
         "tools/block_cache_analyzer/block_cache_trace_analyzer.cc",
         "tools/db_bench_tool.cc",
+        "tools/tool_hooks.cc",
```

Comment:
This is a generated file. Its updated by the release_to_fbcode script for the monthly release. You don't need to do it explicitly.